src/main/webapp/content/scss/themes/theme-dark.scss

```diff
@@ -117,6 +117,47 @@ html {
     border-color: $gray-300;
 }
 
+.chat-widget {
+    background-color: rgb(24, 26, 24) !important;
+}
+
+.chat-header {
+    background-color: var(--artemis-dark) !important;
+}
+
+.chat-body {
+    border: 1px solid rgb(24, 26, 24) !important;
+}
+
+.chat-body::-webkit-scrollbar {
+    background-color: #262626 !important;
+    width: 10px !important;
+}
+
+.chat-body::-webkit-scrollbar-thumb {
+    background-color: #363735 !important;
+}
+
+.client-chat {
+    color: white !important;
+    background-color: #363735 !important;
+}
+
+.my-chat {
+    color: white !important;
+    background-color: #357a38 !important;
+}
+
+.chat-input {
+    background-color: #262626 !important;
+}
+
+.chat-input input {
+    color: white !important;
+    background-color: #363735 !important;
+    border: 1px solid dimgray !important;
+}
+
```

Comment:
Please do not hardcode colors. Use predefined default colors instead; only specify your own colors if strictly necessary. Please follow the guidelines on client theming: https://docs.artemis.cit.tum.de/dev/guidelines/client-design/

Why do you specify component specific styles in the global theme style sheet at all?
If you want to have different colors for both themes, please use CSS variables. There is a section on how to specify colors for each theme in the linked documentation above, so please give the entire page a good read.

I think this entire section should be removed from this file.

Reply:
Done